Engine/AI/DebugDrawGL.cpp

```diff
@@ -0,0 +1,126 @@
+#include "DebugDrawGL.h"
+
+#include "Main/Application.h"
+#include "Module/ModuleProgram.h"
+#include "Module/ModuleDebug.h"
+
+void DebugDrawGL::DepthMask(bool state)
+{
+	glDepthMask(state ? GL_TRUE : GL_FALSE);
+}
+
+void DebugDrawGL::Texture(bool state)
+{
+	if (state)
+	{
+		glEnable(GL_TEXTURE_2D);
+		g_tex.Bind();
+	}
+	else
+	{
+		glDisable(GL_TEXTURE_2D);
+	}
+}
+
+void DebugDrawGL::Begin(DuDebugDrawPrimitives prim, float size)
```

Comment:
This comments goes for the whole file.

Because we are not using lots of this functions we should keep this class as short and clean as posible.